src/Table/Table.tsx

```diff
@@ -43,6 +43,7 @@ export interface Column {
     label: string | React.ReactNode;
     accessor: string;
     canSort?: boolean;
+    cellTemplate?: (row: TableRow, cell: Cell) => React.ReactNode;
```

Comment:
I don't see the need for this frankly speaking. Just Adding  a type reactNode to the value property should suffice. I don't think need a new property.

Reply:
to be honest, filtering by using another hidden columns is really just a workaround. I think we should keep this.

Reply:
This PR address filtering or just rendering reactElement ?

Reply:
I agree with @DanSallau here. It's not just here, we use the same approach will all our components. You only need a type change for `value` prop to `ReactNode` and that's should suffice

Reply:
loop each `TableRow` then each `Cell` to set the value for each cell? Please reconsider.